lib/uklock/mutex.c

```diff
@@ -1,8 +1,69 @@
 #include <uk/mutex.h>
 
+#ifdef CONFIG_LIBUKLOCK_MUTEX_METRICS
+#include <uk/init.h>
+#include <uk/store.h>
```

Comment:
Remove `#include <uk/store.h>`. The linking to `ukstore` can be introduced as another PR